packages/ckeditor5-utils/src/dom/emittermixin.js

```diff
@@ -187,10 +188,15 @@ extend( ProxyEmitter.prototype, EmitterMixin, {
 			return;
 		}
 
-		const domListener = this._createDomListener( event, !!options.useCapture );
+		const listenerOptions = options.useCapture || options.usePassive ? {
```

Comment:
Is there any reason why we cannot pass `listenerOptions = { capture: false, passive: false }` to `addEventListener()`? That would certainly simplify the code and I highly doubt it would have an impact on performance etc..

Reply:
Changed.